independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/options/BootstrapMavenOptions.java

```diff
@@ -90,7 +90,14 @@ public String getOptionValue(String name) {
 
     public String[] getOptionValues(String name) {
         final Object o = options.get(name);
-        return o == null ? null : (String[]) o;
+        if (o == null) {
+            return null;
+        } else if (o instanceof String[]) {
+            return (String[]) o;
+        } else if (o instanceof String) {
+            return new String[]{(String)o};
```

Comment:
I didn't perform a `((String) o).split(",");` here because the code that uses the returned array does that already, but I am wondering if it should be done here in addition...